SU-HSDO/suhumsci
# feat(STN-396):  Traditional Theme Design Polish

# READY FOR REVIEW

## Summary
Editorial polish for Traditional Theme.

Paired with a designer on all changes.

Browser Testing:


## Steps to Test
1.`npm run test` passes.
2. Check for the following on the Traditional Theme:
- [x] All fonts have font-antialiasing now (which only works for Mac OS). Do a broad overview of pages and make sure nothing looks weird or too small now.
- [x] All traditional headings have been modified. Use the "[QA Real Econ Content](http://economics.suhumsci.loc/qa/qa-real-econ-content)" page to check these out -- and add some h5s and h6s.
<img width="1146" alt="Screen Shot 2020-05-27 at 5 00 59 PM" src="https://user-images.githubusercontent.com/12848168/83072457-c877d580-a03c-11ea-8827-15481fdbdca2.png">
<img width="1405" alt="Screen Shot 2020-05-27 at 5 01 09 PM" src="https://user-images.githubusercontent.com/12848168/83072470-d0377a00-a03c-11ea-911d-075bce390603.png">

- [x] Fixes spacing around images in the WYSIWYG to use theme specific spacing variables. Check http://economics.suhumsci.loc/qa/qa-image-styles. 
<img width="1037" alt="Screen Shot 2020-05-27 at 5 01 58 PM" src="https://user-images.githubusercontent.com/12848168/83072478-d6c5f180-a03c-11ea-83ba-5ce895879d82.png">

   - [x] Also fixes an issue that was present with centered images in the WYSIWYG where there was no padding below the caption.
- [x] 3 column layout: check that with 1 sidebar, with 2 sidebars, and no sidebars, that this layout still looks ok at all breakpoints. (Check for colorful theme too)
<img width="1203" alt="Screen Shot 2020-05-27 at 5 03 42 PM" src="https://user-images.githubusercontent.com/12848168/83072544-ed6c4880-a03c-11ea-9d2d-c3d8012582bf.png">

- [x] 3 column with image layout: check that with 1 sidebar, 2 sidebars, and no sidebars, that this layout still looks ok at all breakpoints. (Check for colorful theme too)
<img width="1197" alt="Screen Shot 2020-05-27 at 5 02 22 PM" src="https://user-images.githubusercontent.com/12848168/83072554-f4935680-a03c-11ea-93d4-5ca6677339da.png">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
